domain-model-assistant/Assets/Components/Scripts/Diagram.cs

```diff
@@ -96,46 +101,59 @@ void Start()
     targetOrtho = CanvasScaler.scaleFactor;
     this.raycaster = GetComponent<GraphicRaycaster>();
     GetRequest(GetCdmEndpoint);
+    
+    /* FOR UNITY FRONTEND DEVELOPMENT ONLY ie NO-BACKEND-SERVER*/
+    LoadData();
+    //---------------------------------------//
   }
 
   // Update is called once per frame
   void Update()
   {
-    if ((_currentMode == CanvasMode.Default && InputExtender.MouseExtender.IsDoubleClick()) ||
-        (_currentMode == CanvasMode.AddingClass && InputExtender.MouseExtender.IsSingleClick()))
-    {
-      AddClass("Class" + compartmentedRectangles.Count, Input.mousePosition);
-      ActivateDefaultMode();
-    }
-
-    Zoom();
-
-    if (UseWebcore && _updateNeeded)
-    {
-      if (_getRequestAsyncOp != null && _getRequestAsyncOp.isDone)
-      {
-        var req = _getRequestAsyncOp.webRequest;
-        if (req.downloadHandler != null && !ReferenceEquals(req.downloadHandler, null))
-        {
-          var newResult = req.downloadHandler.text;
-          if (newResult != _getResult)
-          {
-            LoadJson(newResult);
-            _getResult = newResult;
-          }
-        }
-        _updateNeeded = false;
-        req.Dispose();
-      }
-      if (_postRequestAsyncOp != null && _postRequestAsyncOp.isDone)
-      {
-        _postRequestAsyncOp.webRequest.Dispose(); 
-      }
-      if (_deleteRequestAsyncOp != null && _deleteRequestAsyncOp.isDone)
-      {
-        _deleteRequestAsyncOp.webRequest.Dispose();
-      }
-    }
+    // if ((_currentMode == CanvasMode.Default && InputExtender.MouseExtender.IsDoubleClick()) ||
```

Comment:
Why are we commenting this update function out?

Reply:
update function is commented out since the mock server API requests do not work in my unity editor as of (2021-10-17). If your editor works fine, case 1 and case 2 will work for you.<br>
1. `Case 1 ` - Unity Editor Server not working . To allow me to work on the attribute feature the alternate approach was to focus on only reading in the JSON file from the CDM file Gunter shared (ie the JSON file with 1CLASSWITHATTRIBUTE) via the `loadData()` function and display the corresponding attribute in unity editor. 
2. `Case 2` - Unity editor server working. If your unity server works then you will need to modify the mock server to reflect a JSON object with an attribute. Similar to 1CLASSWITHATTRIBUTE, then you can comment the loadData() function and uncomment the update method. It should function the same.